### PR TITLE
HW01: solutions by Volovenko

### PR DESCRIPTION
**Pull request by Volovenko**

### CODE REVIEW

File: HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_02.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def right_equation(number)
+  puts 'Correct' if number == 1 / 3 * (6 * 4)
```
Comment (vladhilko) [inline HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_02.rb]: Does it work? 🤔

Comment (Volovenko) [reply HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_02.rb]: sorry :)
fixed

File: HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def calculate_angle(hours, minutes)
+  # one hour = 360/12 = 30 degrees
+  hours_angle = hours * 30 + minutes * 0.5
+  minutes_angle = 360 * minutes / 60
```
Comment (vladhilko) [inline HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb]: You added a comment for one hour, why don't you want to add the same comment for one minute? 🤔To be consistent

Comment (Volovenko) [reply HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb]: add comment

File: HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+def calculate_angle(hours, minutes)
+  # one hour = 360/12 = 30 degrees
+  hours_angle = hours * 30 + minutes * 0.5
+  # one minute = 360/60 = 6 degrees
+  minutes_angle = 6 * minutes
```
Comment (vladhilko) [inline HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb]: we usually use a different order
```suggestion
  minutes_angle = minutes * 6
```

**Pull request merged**